.gitignore

```diff
@@ -1,10 +1,33 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Why are diffs from my PR showing up here?

Reply:
Nevermind. I see. I thought my changes were merged to master. Makes sense, now.

Reply:
not ideal^^